### PR TITLE
VMO-6785 MCQ SetContactProperty, Multiple Choice and Boolean

### PR DESCRIPTION
**Pull request by object-required**

### CODE REVIEW

File: src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue [hunk 1]
```diff
@@ -85,9 +106,32 @@ export const SelectOneResponseBlockContactPropertyEditor = {
...
+      ]
+    },
+
+    booleanChoiceValueOptions() {
+      return [
+        {value: '', description: this.trans('flow-builder.no-value')},
+        {value: 'TRUE', description: this.trans('flow-builder.true')},
+        {value: 'FALSE', description: this.trans('flow-builder.false')},
+      ]
+    },
```
Comment (bulatgab) [inline src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue:134]: I wonder if we should fail validation when 'No value' is selected (for both boolean and multiple-choice).
Did you see anything about it in the AC/flowcharts? (If not - would you like to ask Nancy?)

File: src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue [hunk 2]
```diff
@@ -150,16 +120,16 @@ export class GenericContactPropertyEditor extends mixins(Lang) {
...
+    if (this.isBlockInteractive(this.block)) {
+      // interactive blocks will have `Entry from this block` option by default
+      this.propertyValue = BLOCK_RESPONSE_EXPRESSION
+    } else {
+      // non interactive blocks will have `Expression` option by default
+      this.propertyValue = EMPTY_STRING_EXPRESSION
     }
+
     this.initPropertyValueAction()
   }
```
Comment (bulatgab) [inline src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue:134]: Could you comment a bit on the changes in this file, please? 😉

**Review by bulatgab — CHANGES_REQUESTED** [review]
Thanks for this PR 👍
(Left two comments asking to clarify things a bit)

File: src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue (hunk 2 above)
Comment (bulatgab) [reply src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue:134]: I mean, I'm afraid that removing stuff from the template will affect other blocks that use GenericContactPropertyEditor, like the Numeric block, won't it?
Screen Shot 2022-09-02 at 12 55 27 PM

**Review by bulatgab — APPROVED** [review]
LGTM 👍

File: src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue (hunk 1 above)
Comment (object-required) [reply src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue:135]: Asked the question in [VMO-6785](https://viamoinc.atlassian.net/browse/VMO-6785?focusedCommentId=21058)

**Pull request merged**